src/vision.cpp

```diff
@@ -124,6 +125,19 @@ bool Vision::configure()
     node_->set_parameter(rclcpp::Parameter(FRAME_ID_PARAM, frame_id_));
   }
 
+  node_->declare_parameter<double>(MAX_PUB_RATE_PARAM);
+  if (!node_->get_parameter<double>(MAX_PUB_RATE_PARAM, max_pub_rate_hz_))
+  {
+    max_pub_rate_hz_ = 30.0;
+    RCLCPP_WARN(node_->get_logger(), "No maximum publication rate set, using max_pub_rate '%f'", max_pub_rate_hz_);
+    node_->set_parameter(rclcpp::Parameter(MAX_PUB_RATE_PARAM, max_pub_rate_hz_));
+  }
+  else
+  {
+    RCLCPP_WARN(node_->get_logger(), "Should have successfully set max_pub_rate: '%f'", max_pub_rate_hz_);
```

Comment:
```suggestion
    RCLCPP_WARN(node_->get_logger(), "Using max_pub_rate: '%f'", max_pub_rate_hz_);
```

Reply:
Oh, good catch. Had this in here for debugging